client/src/PlayerLanding/index.tsx

```diff
@@ -47,8 +47,10 @@ function PlayerLanding() {
       </>
     );
   }
-
-  if (Object.keys(game.roleBag ?? {}).length === 0) {
```

Comment:
@vermentor why was this here?  This file is by far the easiest one to break things, so 🤷‍♂️ but I am not sure why we'd be in a situation where the role bag is empty, and a player is here.  And it is preventing players from rejoining a game when they have a role.  In fact, I am not sure how it's not completely broken

Reply:
My guess is that when you added acknowledge role, you didn't add it to test game creator and so maybe this doesn't get hit in real life?

Reply:
Hmm this was before test game creator.  Role bag is empty until the moment the storyteller starts the game and roles become available to select for players, so that's probably why this existed.  During this time the alternative paths are PlayerRoleSelect (invalid), PlayerRole (invalid), and PlayerInGame (also invalid).

I'd be careful when testing that when a player joins it doesn't take them directly to the PlayerRoleSelect page while seating is still settling.  Although that might actually be fine so long as it brings them back in case of ordering problems.